From 8bc60783f4993d67a0518436f733dba0447ef57c Mon Sep 17 00:00:00 2001
From: Try Omarchy Release <actions@users.noreply.github.com>
Date: Fri, 4 Sep 2026 23:38:20 -0400
Subject: [PATCH] Refresh the guest package lock

Lock Linux 7.2.3, aquamarine 0.15.0 with the matching Hyprland rebuilds, foot 1.28.0, curl 8.22.0, and current archlinux-keyring, GPGME, ImageMagick, and util-linux releases.
---
 guest/packages.lock.json | 52 ++++++++++++++++++++--------------------
 1 file changed, 26 insertions(+), 26 deletions(-)

diff --git a/guest/packages.lock.json b/guest/packages.lock.json
index 3739afd..e9735e8 100644
--- a/guest/packages.lock.json
+++ b/guest/packages.lock.json
@@ -4,7 +4,7 @@
     "abseil-cpp": "20260817.0-2",
     "acl": "2.4.0-1",
     "adwaita-cursors": "50.0-1",
-    "adwaita-fonts": "50.0-1",
+    "adwaita-fonts": "51.0-2",
     "adwaita-icon-theme": "50.0-1",
     "adwaita-icon-theme-legacy": "46.2-3",
     "alsa-card-profiles": "1:1.6.8-1",
@@ -13,8 +13,8 @@
     "alsa-ucm-conf": "1.2.16.1-1",
     "aom": "3.15.0-1",
     "appstream": "1.2.0-1",
-    "aquamarine": "0.14.0-2",
-    "archlinux-keyring": "20260727-1",
+    "aquamarine": "0.15.0-2",
+    "archlinux-keyring": "20260902-1",
     "at-spi2-core": "2.60.6-1",
     "attr": "2.6.0-1",
     "audit": "4.2.1-1",
@@ -39,14 +39,14 @@
     "ca-certificates-utils": "20240618-1",
     "cairo": "1.18.4-1",
     "cfitsio": "1:4.7.0-1",
-    "chromium": "152.0.7977.75-1",
+    "chromium": "152.0.7977.82-1",
     "clang": "22.1.8-1",
     "compiler-rt": "22.1.8-1",
     "coreutils": "9.11-2",
     "cpio": "2.15-3",
     "cpptrace": "1.0.4-2",
     "cryptsetup": "2.8.7-1",
-    "curl": "8.21.0-1",
+    "curl": "8.22.0-1",
     "dav1d": "1.5.4-1",
     "db5.3": "5.3.28-7",
     "dbus": "1.16.2-1",
@@ -80,7 +80,7 @@
     "flac": "1.5.0-1",
     "flex": "2.6.4-6",
     "fontconfig": "2:2.18.3-2",
-    "foot": "1.27.0-2",
+    "foot": "1.28.0-1",
     "freetype2": "2.14.3-1",
     "fribidi": "1.0.16-2",
     "fuse-common": "3.18.2-1",
@@ -118,8 +118,8 @@
     "gnutls": "3.8.13-2",
     "gobject-introspection-runtime": "1.86.0-2",
     "gperftools": "2.18.1-1",
-    "gpgme": "2.1.2-1",
-    "gpgmepp": "2.1.0-1",
+    "gpgme": "2.2.0-1",
+    "gpgmepp": "2.2.0-1",
     "gpm": "1.20.7.r38.ge82d1a6-6",
     "gpu-screen-recorder": "6.1.0-1",
     "graphene": "1.10.8-2",
@@ -156,20 +156,20 @@
     "hwloc": "2.14.0-1",
     "hyprcursor": "0.1.13-7",
     "hyprgraphics": "0.5.1-4",
-    "hypridle": "0.1.8-1",
-    "hyprland": "0.56.2-1",
-    "hyprland-guiutils": "0.2.2-2",
+    "hypridle": "0.1.8-2",
+    "hyprland": "0.56.2-2",
+    "hyprland-guiutils": "0.2.2-3",
     "hyprland-preview-share-picker": "0.2.1-1",
     "hyprlang": "0.6.8-5",
-    "hyprpicker": "0.4.7-3",
+    "hyprpicker": "0.4.7-4",
     "hyprsunset": "0.4.0-3",
-    "hyprtoolkit": "0.5.4-4",
+    "hyprtoolkit": "0.5.4-5",
     "hyprutils": "0.14.1-1",
     "hyprwayland-scanner": "0.4.6-1",
     "hyprwire": "0.3.1-3",
     "iana-etc": "20260617-1",
     "icu": "78.3-1",
-    "imagemagick": "7.1.2.30-1",
+    "imagemagick": "7.1.2.31-1",
     "imath": "3.2.3-1",
     "imv": "5.0.1-2",
     "iniparser": "4.2.6-2",
@@ -225,14 +225,14 @@
     "libcap-ng": "0.9.5-1",
     "libcdio": "2.4.0-1",
     "libcdio-paranoia": "10.2+2.0.2-3",
-    "libcgif": "0.5.3-1",
+    "libcgif": "0.5.4-1",
     "libcloudproviders": "0.4.1-1",
     "libcolord": "1.4.8-1",
     "libcue": "2.3.0-2",
     "libcups": "2:2.4.19-1",
     "libdaemon": "0.14-6",
     "libdatrie": "0.2.14-1",
-    "libde265": "1.1.1-1",
+    "libde265": "1.1.2-1",
     "libdecor": "0.2.5-1",
     "libdeflate": "1.26-1",
     "libdisplay-info": "0.3.0-1",
@@ -299,7 +299,7 @@
     "libnetfilter_conntrack": "1.1.1-1",
     "libnewt": "0.52.25-2",
     "libnfnetlink": "1.0.2-2",
-    "libnftnl": "1.3.1-1",
+    "libnftnl": "1.3.2-1",
     "libnghttp2": "1.70.0-1",
     "libnghttp3": "1.18.0-1",
     "libngtcp2": "1.25.0-1",
@@ -397,7 +397,7 @@
     "libxkbcommon": "1.13.2-1",
     "libxkbcommon-x11": "1.13.2-1",
     "libxkbfile": "1.2.0-1",
-    "libxml2": "2.15.3-1",
+    "libxml2": "2.15.4-1",
     "libxmlb": "0.3.29-1",
     "libxmu": "1.3.1-1",
     "libxpresent": "1.0.2-1",
@@ -414,7 +414,7 @@
     "libzip": "1.11.4-1",
     "licenses": "20240728-1",
     "lilv": "0.28.0-1",
-    "linux": "7.2.2.arch1-1",
+    "linux": "7.2.3.arch1-2",
     "linux-api-headers": "7.2-1",
     "llhttp": "9.3.1-1",
     "llvm-libs": "22.1.8-2",
@@ -424,7 +424,7 @@
     "lua": "5.5.1-1",
     "lua51-lpeg": "1.1.0-5",
     "lua54": "5.4.9-1",
-    "luajit": "2.1.1787165859+1ee778a-1",
+    "luajit": "2.1.1788460057+24c20c9-1",
     "lv2": "1.18.10-2",
     "lz4": "1:1.10.0-2",
     "lzo": "2.10-5",
@@ -451,7 +451,7 @@
     "neovim": "0.12.5-1",
     "nettle": "4.0-1",
     "networkmanager": "1.58.1-1",
-    "nftables": "1:1.1.6-3",
+    "nftables": "1:1.1.7-3",
     "noto-fonts": "1:2026.09.01-1",
     "noto-fonts-emoji": "1:2.051-1",
     "npth": "1.8-1",
@@ -591,12 +591,12 @@
     "unibilium": "2.1.4-1",
     "unzip": "6.0-23",
     "upower": "1.91.3-1",
-    "util-linux": "2.42.2-1",
-    "util-linux-libs": "2.42.2-1",
+    "util-linux": "2.42.3-1",
+    "util-linux-libs": "2.42.3-1",
     "uwsm": "0.26.7-1",
     "v4l-utils": "1.32.0-2",
-    "vapoursynth": "77-2",
-    "vid.stab": "1.1.1-3",
+    "vapoursynth": "79-1",
+    "vid.stab": "1.1.2-1",
     "vmaf": "3.2.0-1",
     "volume_key": "0.3.12-12",
     "vulkan-icd-loader": "1.4.357.0-1",
@@ -624,7 +624,7 @@
     "xcb-util-wm": "0.4.2-2",
     "xdg-desktop-portal": "1.22.1-2",
     "xdg-desktop-portal-gtk": "1.15.3-1",
-    "xdg-desktop-portal-hyprland": "1.4.1-1",
+    "xdg-desktop-portal-hyprland": "1.4.1-2",
     "xdg-terminal-exec": "0.14.3-1",
     "xdg-user-dirs": "0.20-1",
     "xdg-user-dirs-gtk": "0.16-1",
-- 
2.55.0

